src/api/ledger/transaction.js

```diff
@@ -78,11 +78,22 @@ function getTransactionAsync(identifier: string, options: TransactionOptions,
     }
   }
 
+
+  /* eslint-disable no-unused-vars, handle-callback-err */
```

Comment:
We can avoid the disabling comment if we use the technique described here to selectively mark individual parameters as unused: https://github.com/ripple/javascript-style-guide/pull/20

Also, why is the error case not being handled?